scripts/pythondistdeps.py

```diff
@@ -143,10 +201,30 @@ def convert(name, operator, version_id):
 
 def normalize_name(name):
     """https://www.python.org/dev/peps/pep-0503/#normalized-names"""
-    import re
```

Comment:
Why is this no longer lazy? Because the performance impact was negligible?

Reply:
Indeed